k8s/manifests/charts/ck-loadbalancer/values.yaml

```diff
@@ -1,3 +1,4 @@
+driver: cilium
```

Comment:
this should rather be set on the helm chart values in cilium, just as we set this on metallb as well.

Reply:
This is just a default value, it's overridden in the Go code.

Reply:
yes. implementations (cilium or metallb) must specify this, let's not hide this requirement with implicit defaults